### DISCUSSION TITLE
R-Precision using pyserini.eval.trec_eval

### DISCUSSION THREAD

**Discussion by Fatima-Haouari** [castorini/pyserini#942] [Q&A] [answered]
Hi,
I managed to evaluate my runs using Pyserini. However, I am not able to find out how to get the R-Precision.
Kindly direct me if there is any documentation for the evaluation measures supported by Pyserini.

**Comment by lintool** [comment, marked as answer]
As previously mentioned: https://github.com/castorini/pyserini/discussions/932#discussioncomment-1949447

> For future reference, please file questions under "Discussions" - Q&A.

Pyserini just passes options to `trec_eval` - see complete documentation here: https://github.com/usnistgov/trec_eval

**Reply by Fatima-Haouari** [reply]
Solved. Thanks for your help.
